index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<header>
+<link rel="stylesheet" type"text/css" href="css/style.css">
+</header>
+<Body>
+<table>
+	<tbody>
+		<tr>
+			<td width="33%">
+				<button id="increase">Increase</button>
+			</td>
+			<td width="33%">
+				<button	id="decrease">Decrease</button>
+			</td>
+			<td width="33%">
+				<button id="reset">Reset</button>
+			</td>
+		</tr>
+	</tbody>
+<p class="display">0</p>
```

Comment:
No need to have a value in your HTML and then separately declare a `counter` variable in your JS. You want to pass the `counter` variable from your model into your view when the page is loaded, the same way you would after any button is pressed.